common/clickhouse-srv/src/types/block/mod.rs

```diff
@@ -240,12 +240,14 @@ impl<K: ColumnType> Block<K> {
     }
 
     /// Add new column into this block
+    #[must_use]
```

Comment:
What's the `[must_use]` attribute for

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#return_self_not_must_use

> Methods returning Self often create new values, having the #[must_use] attribute prevents users from “forgetting” to use the newly created value.